package.json

```diff
@@ -33,6 +33,7 @@
   },
   "dependencies": {
     "bn.js": "4.11.6",
+    "moment": "^2.22.2",
```

Comment:
Let's use [date-fns@2.0.0-alpha.7](https://date-fns.org/v2.0.0-alpha.7/docs/Getting-Started) for now since that's what our other frontend projects are using.

[moment.js is yuge](https://github.com/you-dont-need/You-Dont-Need-Momentjs).